File: Data Structures & Algorithms/find-duplicate-integer/submission-1.cs
public class Solution {
    public int FindDuplicate(int[] nums)
{
    for (int i = 0; i < nums.Length; i++)
    {
        int negativeVal = nums[i] * -1;
        if (nums.Contains(negativeVal))
        {
            return negativeVal * -1;
        }
        nums[i] = negativeVal;

    }

    return 0;
}
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: list[int], target: int) -> list[int]:
        dict_ = {}
        indexes = []

        for index, num in enumerate(nums):
            target_num = target - num
            if target_num in dict_:
                indexes.append(dict_.get(target_num))
                indexes.append(index)
                return indexes
            dict_[num] = index


solution = Solution()
print(solution.twoSum([4,5,6], 10))
print(solution.twoSum([5,5], 10))